.github/workflows/update-deps.yml

```diff
@@ -0,0 +1,63 @@
+name: Tools and deps update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
+
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - .github/workflows/update-deps.yml
```

Comment:
Maybe I misunderstand but don't we want to do this only before a deploy on `main`?

Reply:
Hum I think we only want it after each Uppy release.

Reply:
But now it runs on every PR?

Reply:
maybe if we could have uppy publish to npm (uppy@latest) on every commit, then we could use that here so we could update uppy.io to uppy@latest using the workflow_dispatch whenever we need that (or the schedule will)

Reply:
> But now it runs on every PR?

It runs on every PRs that modify `.github/workflows/update-deps.yml`.